includes/Services/FlowService.php

```diff
@@ -0,0 +1,262 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+/**
+ * Class FlowService
+ */
+class FlowService {
+
+	/**
+	 * Initialize Flow Data on refresh.
+	 *
+	 * @return boolean
+	 */
+	public static function initalize_flow_data() {
+		$default_flow_data = self::get_default_flow_data();
+		$flow_data         = self::read_flow_data_from_wp_option();
+
+		if ( ! ( $flow_data ) ) {
+			return \update_option( Options::get_option_name( 'flow' ), $default_flow_data );
+		}
+
+		if ( ! isset( $flow_data['version'] ) || strcmp( $flow_data['version'], $default_flow_data['version'] ) !== 0 ) {
+			$flow_data_upgrades = NFD_ONBOARDING_DIR . '\\includes\\Data\\Flows\\Upgrades\\' . $default_flow_data['version'] . '.php';
```

Comment:
Hey, do you know if this code is windows specific? I think this won't work on Mac and Linux, and as the site will be running on a Linux env, maybe we can have a more generalized way ?
<img width="728" alt="Screenshot 2023-05-11 at 7 04 41 PM" src="https://github.com/newfold-labs/wp-module-onboarding/assets/48691514/a8071ce5-5c80-4b9b-9664-7a18467e5f9a">